### PR TITLE
Query update animals - Add animals, update animals with transactions

### PR DESCRIPTION
**Pull request by Danie12345**
In this PR I:

- Added the `species` column to the `animals` table. image
- Inserted new data into the `animals` table. image Initially, these animals have positive weights as I believed it to be an error, I later modified them once I noticed it was intentional.
- Set everything to unspecified then rolled back.
- Added the species in a transaction and committed. image
- Deleted everything in a transaction, then rolled back. image
- In a transaction: deleted all animals born after Jan 1st, 2022, created a savepoint, multiplied all animals' weights by -1, rolled back to the previous savepoint, update any negative weights to be multiplied by -1, then committed. image
- Added the exercise queries:
  - How many animals are there?:
   image
  - How many animals have never tried to escape?:
   image
  - What is the average weight of animals?:
   image
  - Who escapes the most, neutered or not neutered animals?: image
   True refers to neutered animals.
  - What is the minimum and maximum weight of each type of animal?: image
  - What is the average number of escape attempts per animal type of those born between 1990 and 2000?: image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /*Queries that provide answers to the questions from all projects.*/
 
+--  project0
+
 select * from animals where name like '%mon';
```
Comment (BenMukebo) [inline queries.sql:6]: - [ ] Kindly make sure to address this 👇🏻 
- Inside a transaction update the `animals` table by setting the `species` column to `unspecified`. Verify that change was made. Then roll back the change and verify that the `species` columns went back to the state before the transaction.

File: schema.sql [hunk 2]
```diff
@@ -8,4 +10,44 @@ create table animals(
...
+where date_of_birth > '2022-01-01';
+savepoint sp0;
+update animals
+set weight_kg = weight_kg * (-1);
+rollback to sp0;
+update animals
+set weight_kg = weight_kg * (-1)
+where weight_kg < 0;
+commit;
+select * from animals;
```
Comment (BenMukebo) [inline schema.sql]: - [ ] Make sure Everything that is a query that helps to analyze data should be saved in the `queries.sql` file and not in the `schema.sql` file, Kindly remove it from here that can be only written in the `queries.sql` file 👍🏻

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Danie12345,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻



## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @Danie12345, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**